### PR TITLE
cuitlazotlac/francais

### PR DESCRIPTION
**Pull request by cuitlazotlac**
## Summary of changes

_Description of the changes included in this pull request. 
Steps to follow to confirm the feature is completed or the bug/issue is resolved. 
Are there any dependencies? 
Does this code break any existing functionality or introduce any risk?_

## Relevant Context

- Feature: [ticket number](link to project management ticket)
- Relevant Pull Request: # 

## Further comments

--- 
## Types of changes

- [x] Content edit, File upload, content addition
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Technical Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by andytudhope** [comment]
Bonjour @cuitlazotlac :heart: 

I merged this because it was getting too big to readily keep track of. If you would finish the translations for the Research and Development pages, I will merge this into production :pray:

File: content/fr/resources/index.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Vivez dans les strates
+
+L'un des aspects les plus intéressants des personnes que nous avons rencontrées dans le web3 est la nature interdisciplinaire de bon nombre de leurs intérêts. Personne n'est qu'un designer, ou seulement un développeur, ou qui participe à tout cela à cause d'une seule idée ou d'un seul produit. Bien que beaucoup d'entre nous aient une expertise particulière, le but des outils utiles devrait être de nous aider [à poser toujours des meilleurs questions](/learn/module-2/better-questions) sur nous-mêmes, notre environnement et les mondes que nous créons ensemble.
+
+Connaître des personnes aussi intéressantes et intéressées est libérateur, même si cela rend la catégorisation plus difficile. Nous ne souhaitons pas vous enfermer dans les catégories présentées ici : chacune est liée à chacune des autres, et nous vous encourageons à explorer toutes les sections et à réfléchir à la manière dont vous pouvez utiliser les outils placés dans chacune pour [jouer avec les limites] (/build/games/infinite) entre chaque champ.
```
Comment (anthonygraignic) [inline content/fr/resources/index.mdx:26]: Markdown link is broken (Did Deepl broke it :p ?)

File: content/fr/resources/index.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+header: true
+headerOrder: 5
+title: Resources
+description: An overview of the tools we use and recommend to help you on your journey towards a better web.
```
Comment (anthonygraignic) [inline content/fr/resources/index.mdx:5]: Need to be translated too

File: content/fr/resources/design-ux.mdx [hunk 3]
```diff
@@ -0,0 +1,229 @@
+---
+title: Design & UX
+order: 1
+hideLanguageSelector: true
+description: A highly curated list of design and UX tools made or recommended by our community.
```
Comment (anthonygraignic) [inline content/fr/resources/design-ux.mdx:5]: Need to be translated too

File: content/fr/resources/design-ux.mdx [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  to="https://design-system.mkr-js-prod.now.sh/recipes"
+  sx={{ alignSelf: 'flex-start', mt: 'auto' }}>
+  Craft
+</Button>
+
+</Flex>
+
+</Column>
+
+## Space Design
```
Comment (anthonygraignic) [inline content/fr/resources/design-ux.mdx:191]: Need to be translated too ?

File: content/fr/resources/design-ux.mdx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+  Record
+</Button>
+
+</Flex>
+
+</Chocolate>
+
+</Box>
+
+## System Design
```
Comment (anthonygraignic) [inline content/fr/resources/design-ux.mdx:165]: Need to be translated too ?

File: content/fr/resources/development.mdx [hunk 6]
```diff
@@ -0,0 +1,191 @@
+---
+title: Development
```
Comment (anthonygraignic) [inline content/fr/resources/development.mdx:2]: Whole page need to be translated

File: content/fr/resources/security.mdx [hunk 7]
```diff
@@ -0,0 +1,203 @@
+---
+title: Security
+order: 3
+hideLanguageSelector: true
+description: A highly curated collection of the best smart contract security tools currently available.
```
Comment (anthonygraignic) [inline content/fr/resources/security.mdx:5]: Need to be translated

File: content/fr/resources/security.mdx [hunk 8]
```diff
@@ -0,0 +1,203 @@
+---
+title: Security
```
Comment (anthonygraignic) [inline content/fr/resources/security.mdx:2]: Need to be translated

**Review by anthonygraignic — COMMENTED** [review]
Great translation 🙌️, there is just one not-translated page remaining and page descriptions are not translated.

**Pull request merged**